backend/src/main/java/corea/auth/service/LoginService.java

```diff
@@ -37,6 +39,14 @@ public String createAccessToken(Member member) {
 
     @Transactional
     public String publishRefreshToken(Member member) {
+
+        long memberId = member.getId();
+        Optional<LoginInfo> loginInfo = loginInfoRepository.findByMemberId(memberId);
```

Comment:
```suggestion
    @Transactional
    public String publishRefreshToken(Member member) {
        long memberId = member.getId();
        return loginInfoRepository.findByMemberId(memberId)
                .orElseGet(() -> saveLoginInfo(member))
                .getRefreshToken();
    }
    private LoginInfo saveLoginInfo(Member member) {
        return loginInfoRepository.save(new LoginInfo(member, tokenProvider.createToken(member, tokenProperties.expiration();
    }
```

불필요한 Optional 선언 + isPresent 구문 제거 가능은 할 거 같슴당